### PR TITLE
fix: Remove aria-label for breadcrumb items(CXSPA-1122)

### PR DESCRIPTION
**Pull request by kimhw0630**
Closes: https://jira.tools.sap/browse/CXSPA-1122

### CODE REVIEW

**Review by Zeyber — APPROVED** [review]
Please change the commit message to `fix: xxxxxxxx` before merging this is not a new feature and its important to distinguish this for docs and code freeze.

Also with a11y fixes/features, please wait for Miguel's approval too before merging. We first do code checks as developers and then pass tickets to Miguel for his approval after dev approval. Just in case you weren't already aware ;)

**Review by developpeurweb — APPROVED** [review]

**Pull request merged**
